File: 1116.py
n = int(input())
d = 0.0
for i in range(n):
    a,b = map(int, input().split())
    if(b==0):
        print("divisao impossivel")
    elif(b!=0):
        d = a/b
        print("%.1f" %d)

    

File: 1116.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    double n, x, y;
    cin >> n;
    while(n--){
        cin >> x >> y;
        if(y==0){
            cout << "divisao impossivel\n";
        }else{
            cout << fixed << setprecision(1) << x/y << "\n";
        }
        
    }

}